distribution/src/resources/drill-override-example.conf

```diff
@@ -133,6 +133,8 @@ drill.exec: {
   security.user.auth {
     enabled: false,
     packages += "org.apache.drill.exec.rpc.user.security",
+    # There are 2 implementations available "pam" using JPAM
+    # and "pam4j" using libpam4j
```

Comment:
As discussed in person, perhaps explain how the `impl` property here related to the implementation class. (Evidently via an annotation. So, maybe just name the annotation used.)

Reply:
Updated the comment here to reflect the Annotation name `UserAuthenticatorTemplate`